packages/components/src/toggle-group-control/toggle-group-control/toggle-group-control-backdrop.tsx

```diff
@@ -1,86 +1,96 @@
+/**
+ * External dependencies
+ */
+// eslint-disable-next-line no-restricted-imports
+import { useReducedMotion } from 'framer-motion';
+
 /**
  * WordPress dependencies
  */
-import { useState, useEffect, memo } from '@wordpress/element';
+import { useLayoutEffect, useState, memo } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import type { ToggleGroupControlBackdropProps } from '../types';
-import { BackdropView } from './styles';
+import { useToggleGroupControlContext } from '../context';
+import { CONFIG } from '../../utils';
+import { AnimatedBackdrop } from './styles';
+
+const TRANSITION_CONFIG = {
+	type: 'spring',
+	bounce: 0,
+	// Transition durations in the config are expressed as a string in milliseconds,
+	// while `framer-motion` needs them as integers in seconds.
+	duration: parseInt( CONFIG.transitionDurationSlow, 10 ) / 1000,
```

Comment:
What do you think about just hardcoding the duration value here? I guess because:

- It's a pretty specialized animation anyway
- What if the `CONFIG` values are ever changed from `ms` to `s`
- It seems like overkill to extract a separate `TRANSITION_DURATION_MS` object (Or maybe not? Will it come up a lot in these JS animations?)

No strong opinion though, just throwing it out there.

Reply:
I'm happy to hardcode it, although we may want to reconsider having a more clear, better organised shared "transition duration" settings when we start refactoring our shared configs